### PR TITLE
add ch benchmark

### PR DESCRIPTION
**Pull request by dbsid**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: benchmark/benchmark-tidb-using-ch.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
+---
+title: 如何对 TiDB 进行 CH 测试
```
Comment (qiancai) [inline benchmark/benchmark-tidb-using-ch.md:2]:
```suggestion
title: 如何对 TiDB 进行 CH 测试
summary: 本文介绍如何对 TiDB 进行 CH 测试。
```

File: benchmark/benchmark-tidb-using-ch.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+---
+
+# 如何对 TiDB 进行 CH 测试
+
+本文介绍如何对 TiDB 进行 CH Benchmark 测试。
```
Comment (qiancai) [inline benchmark/benchmark-tidb-using-ch.md:7]: CH Benchmark 麻烦可以给一个参考链接吗

File: benchmark/benchmark-tidb-using-ch.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+
+CH Benchmark 是包含 [TPC-C](http://www.tpc.org/tpcc/) 和 [TPC-H](http://www.tpc.org/tpch/) 的混合负载，同用来测试 HTAP 系统的最常见负载。TiDB 推荐部署 TiFlash 组件运行 CH Benchmark，对于 TPC-C 联机交易数据，系统自动实时同步到 TiFlash 组件；TiDB 优化器会自动将 TPCH 负载的 OLAP 查询下推到 TiFflash MPP 引擎进行高效执行。
```
Comment (qiancai) [inline benchmark/benchmark-tidb-using-ch.md:9]:
```suggestion
CH Benchmark 包含 [TPC-C](http://www.tpc.org/tpcc/) 和 [TPC-H](http://www.tpc.org/tpch/) 的混合负载，是用来测试 HTAP 系统的最常见负载。

在进行 CH Benchmark 测试前，你需要部署 TiDB 的 HTAP 组件 [TiFlash](/tiflash/tiflash-overview.md)。部署后，对于 TPC-C 联机交易数据，系统自动实时同步数据到 TiFlash 组件；TiDB 优化器会自动将 TPC-H 负载的 OLAP 查询下推到 TiFlash MPP 引擎进行高效执行。
```

File: benchmark/benchmark-tidb-using-ch.md [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+
+我们将以 1000 WAREHOUSE 为例进行测试。
+
```
Comment (qiancai) [inline benchmark/benchmark-tidb-using-ch.md:15]:
```suggestion
```

File: benchmark/benchmark-tidb-using-ch.md [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+```shell
+tiup install bench
+```
+
+关于 TiUP Bench 组件的详细用法可参考 [TiUP Bench](/tiup/tiup-bench.md)。
+
+## 导入数据
+
+### 导入 TPC-C 数据
```
Comment (qiancai) [inline benchmark/benchmark-tidb-using-ch.md:28]:
```suggestion
### 导入 TPC-C 数据

```

File: benchmark/benchmark-tidb-using-ch.md [hunk 6]
```diff
@@ -0,0 +1,164 @@
...
+generating region table
+generate region table done
+generating suppliers table
+generate suppliers table done
+creating view revenue1
+```
+
+## 同步数据到 TiFlash
+
+如果 TiDB 集群部署了 TiFlash，需要同步数据到 TiFLash
```
Comment (qiancai) [inline benchmark/benchmark-tidb-using-ch.md:69]:
```suggestion
部署 TiFlash 后，TiFlash 并不会自动同步 TiKV 数据，你需要执行以下 SQL 语句将数据同步到 TiFlash。同步后，TiDB 将创建对应的 TiFlash 副本。

```

File: benchmark/benchmark-tidb-using-ch.md [hunk 7]
```diff
@@ -0,0 +1,164 @@
...
+alter table orders set tiflash replica 2;
+alter table stock set tiflash replica 2;
+alter table warehouse set tiflash replica 2;
+alter table nation set tiflash replica 2;
+alter table region set tiflash replica 2;
+alter table supplier set tiflash replica 2;
+```
+
+## 搜集统计信息
+
```
Comment (qiancai) [inline benchmark/benchmark-tidb-using-ch.md:86]: 这里可以加一句介绍的话吗？为啥要执行这个步骤

File: benchmark/benchmark-tidb-using-ch.md [hunk 8]
```diff
@@ -0,0 +1,164 @@
...
+analyze table stock;
+analyze table warehouse;
+analyze table nation;
+analyze table region;
+analyze table supplier
+```
+
+## 运行测试
+
+以 50 TP并发，1 AP 并发为例，运行测试的命令是：
```
Comment (qiancai) [inline benchmark/benchmark-tidb-using-ch.md:105]:
```suggestion
以 50 TP 并发，1 AP 并发为例，运行以下测试命令：
```

File: benchmark/benchmark-tidb-using-ch.md [hunk 9]
```diff
@@ -0,0 +1,164 @@
...
+
+
+{{< copyable "shell-regular" >}}
+```shell
+go-tpc ch --host 172.16.5.140 -P4000 --warehouses 1000 run -D tpcc -T 50 -t 1 --time 1h
+```
+
+运行过程中控制台上会持续打印测试结果：
+
```
Comment (qiancai) [inline benchmark/benchmark-tidb-using-ch.md:114]: 下面的结果还可以省略一下吗，可能还是有点长

File: benchmark/benchmark-tidb-using-ch.md [hunk 10]
```diff
@@ -0,0 +1,164 @@
...
+[Current] NEW_ORDER - Takes(s): 10.0, Count: 13238, TPM: 79429.5, Sum(ms): 1010795.3, Avg(ms): 76.4, 50th(ms): 75.5, 90th(ms): 104.9, 95th(ms): 117.4, 99th(ms): 159.4, 99.9th(ms): 234.9, Max(ms): 352.3
+[Current] ORDER_STATUS - Takes(s): 10.0, Count: 1224, TPM: 7350.6, Sum(ms): 17874.1, Avg(ms): 14.6, 50th(ms): 13.6, 90th(ms): 23.1, 95th(ms): 27.3, 99th(ms): 37.7, 99.9th(ms): 54.5, Max(ms): 60.8
+[Current] PAYMENT - Takes(s): 10.0, Count: 12650, TPM: 75901.1, Sum(ms): 761981.3, Avg(ms): 60.3, 50th(ms): 56.6, 90th(ms): 88.1, 95th(ms): 104.9, 99th(ms): 159.4, 99.9th(ms): 218.1, Max(ms): 318.8
+[Current] STOCK_LEVEL - Takes(s): 10.0, Count: 1179, TPM: 7084.9, Sum(ms): 32829.8, Avg(ms): 27.9, 50th(ms): 26.2, 90th(ms): 37.7, 95th(ms): 44.0, 99th(ms): 71.3, 99.9th(ms): 100.7, Max(ms): 117.4
+[Current] Q12    - Count: 1, Sum(ms): 9945.8, Avg(ms): 9944.7
+[Current] Q13    - Count: 1, Sum(ms): 1729.6, Avg(ms): 1729.6
+...
+```
+
+运行结束后，会打印测试统计结果：
```
Comment (qiancai) [inline benchmark/benchmark-tidb-using-ch.md:131]: 下面的结果还可以省略一下吗，可能还是有点长

File: benchmark/benchmark-tidb-using-ch.md (hunk 2 above)
Comment (dbsid) [reply benchmark/benchmark-tidb-using-ch.md:8]: 没有非常官方的，要不要放第二个是 tableau 的？

https://dl.acm.org/doi/10.1145/1988842.1988850
https://research.tableau.com/sites/default/files/a8-cole.pdf

File: benchmark/benchmark-tidb-using-ch.md [hunk 11]
```diff
@@ -0,0 +1,167 @@
...
+tiup install bench
+```
+
+关于 TiUP Bench 组件的详细用法可参考 [TiUP Bench](/tiup/tiup-bench.md)。
+
+## 导入数据
+
+### 导入 TPC-C 数据
+**导入数据通常是整个 TPC-C 测试中最耗时，也是最容易出问题的阶段。**
+
```
Comment (qiancai) [inline benchmark/benchmark-tidb-using-ch.md:30]:
```suggestion

本文将以 1000 WAREHOUSE 为例进行导入和测试。
```

File: benchmark/benchmark-tidb-using-ch.md (hunk 10 above)
Comment (dbsid) [reply benchmark/benchmark-tidb-using-ch.md:132]: 不能省略，都是很重要的内容

File: benchmark/benchmark-tidb-using-ch.md (hunk 2 above)
Comment (dbsid) [reply benchmark/benchmark-tidb-using-ch.md:8]: 选了第二个链接

File: benchmark/benchmark-tidb-using-ch.md [hunk 12]
```diff
@@ -0,0 +1,169 @@
...
+```
+
+## 运行测试
+
+以 50 TP 并发，1 AP 并发为例，运行以下测试命令：
+
+
+{{< copyable "shell-regular" >}}
+```shell
+go-tpc ch --host 172.16.5.140 -P4000 --warehouses 1000 run -D tpcc -T 50 -t 1 --time 1h
```
Comment (shawn0915) [inline benchmark/benchmark-tidb-using-ch.md:112]: 这里建议改成 `tiup bench ch run`, 前后统一。

File: benchmark/benchmark-tidb-using-ch.md [hunk 13]
```diff
@@ -0,0 +1,169 @@
...
+alter table stock set tiflash replica 2;
+alter table warehouse set tiflash replica 2;
+alter table nation set tiflash replica 2;
+alter table region set tiflash replica 2;
+alter table supplier set tiflash replica 2;
+```
+
+## 搜集统计信息
+
+为了确保优化器能产生正确的执行计划，需要提前搜集统计信息。
```
Comment (qiancai) [inline benchmark/benchmark-tidb-using-ch.md:88]: 是“为了提高优化器的生成执行计划的效率”？

File: benchmark/benchmark-tidb-using-ch.md [hunk 14]
```diff
@@ -0,0 +1,169 @@
...
+
+以 50 TP 并发，1 AP 并发为例，运行以下测试命令：
+
+{{< copyable "shell-regular" >}}
+
+```shell
+go-tpc ch --host 172.16.5.140 -P4000 --warehouses 1000 run -D tpcc -T 50 -t 1 --time 1h
+```
+
+运行过程中控制台上会持续打印测试结果：
```
Comment (qiancai) [inline benchmark/benchmark-tidb-using-ch.md:115]:
```suggestion
运行过程中控制台上会持续打印测试结果。例如：
```

File: benchmark/benchmark-tidb-using-ch.md [hunk 15]
```diff
@@ -0,0 +1,169 @@
...
+[Current] NEW_ORDER - Takes(s): 10.0, Count: 13238, TPM: 79429.5, Sum(ms): 1010795.3, Avg(ms): 76.4, 50th(ms): 75.5, 90th(ms): 104.9, 95th(ms): 117.4, 99th(ms): 159.4, 99.9th(ms): 234.9, Max(ms): 352.3
+[Current] ORDER_STATUS - Takes(s): 10.0, Count: 1224, TPM: 7350.6, Sum(ms): 17874.1, Avg(ms): 14.6, 50th(ms): 13.6, 90th(ms): 23.1, 95th(ms): 27.3, 99th(ms): 37.7, 99.9th(ms): 54.5, Max(ms): 60.8
+[Current] PAYMENT - Takes(s): 10.0, Count: 12650, TPM: 75901.1, Sum(ms): 761981.3, Avg(ms): 60.3, 50th(ms): 56.6, 90th(ms): 88.1, 95th(ms): 104.9, 99th(ms): 159.4, 99.9th(ms): 218.1, Max(ms): 318.8
+[Current] STOCK_LEVEL - Takes(s): 10.0, Count: 1179, TPM: 7084.9, Sum(ms): 32829.8, Avg(ms): 27.9, 50th(ms): 26.2, 90th(ms): 37.7, 95th(ms): 44.0, 99th(ms): 71.3, 99.9th(ms): 100.7, Max(ms): 117.4
+[Current] Q12    - Count: 1, Sum(ms): 9945.8, Avg(ms): 9944.7
+[Current] Q13    - Count: 1, Sum(ms): 1729.6, Avg(ms): 1729.6
+...
+```
+
+运行结束后，会打印测试统计结果：
```
Comment (qiancai) [inline benchmark/benchmark-tidb-using-ch.md:133]:
```suggestion
命令运行结束后，控制台会打印测试统计结果：
```

File: benchmark/benchmark-tidb-using-ch.md (hunk 13 above)
Comment (dbsid) [reply benchmark/benchmark-tidb-using-ch.md:88]: 统计信息是优化器产生正确执行计划的前提

File: benchmark/benchmark-tidb-using-ch.md [hunk 16]
```diff
@@ -0,0 +1,169 @@
...
+### 导入 TPC-C 数据
+
+**导入数据通常是整个 TPC-C 测试中最耗时，也是最容易出问题的阶段。**
+
+本文将以 1000 WAREHOUSE 为例进行导入和测试。在 shell 中运行 TiUP 命令：
+
+{{< copyable "shell-regular" >}}
+
+```shell
+tiup bench tpcc -H 172.16.5.140 -P 4000 -D tpcc --warehouses 1000 prepare
```
Comment (lilinghai) [inline benchmark/benchmark-tidb-using-ch.md:35]: 指定一下线程数量？

File: benchmark/benchmark-tidb-using-ch.md [hunk 17]
```diff
@@ -0,0 +1,169 @@
...
+generating region table
+generate region table done
+generating suppliers table
+generate suppliers table done
+creating view revenue1
+```
+
+## 同步数据到 TiFlash
+
+部署 TiFlash 后，TiFlash 并不会自动同步 TiKV 数据，你需要执行以下 SQL 语句创建 TiFlash 副本。创建 TiFlash 副本后，系统自动实时同步最新数据到 TiFlash 组件。以下例子中，集群中部署了两个 TiFlash 节点，将 replica 设置为 2。
```
Comment (lilinghai) [inline benchmark/benchmark-tidb-using-ch.md:69]: tiflash 6.0 极其后续的版本支持 `alter database tpcc set tiflash replica 2` 更方便，可以提一下

File: benchmark/benchmark-tidb-using-ch.md [hunk 18]
```diff
@@ -0,0 +1,169 @@
...
+
+## 运行测试
+
+以 50 TP 并发，1 AP 并发为例，运行以下测试命令：
+
+{{< copyable "shell-regular" >}}
+
+```shell
+go-tpc ch --host 172.16.5.140 -P4000 --warehouses 1000 run -D tpcc -T 50 -t 1 --time 1h
+```
```
Comment (lilinghai) [inline benchmark/benchmark-tidb-using-ch.md:113]: 执行前需要说明一下要等待 tiflash 副本 ready

File: benchmark/benchmark-tidb-using-ch.md [hunk 19]
```diff
@@ -0,0 +1,169 @@
...
+generating nation table
+generate nation table done
+generating region table
+generate region table done
+generating suppliers table
+generate suppliers table done
+creating view revenue1
+```
+
+## 同步数据到 TiFlash
```
Comment (qiancai) [inline benchmark/benchmark-tidb-using-ch.md:67]:
```suggestion
## 创建 TiFlash 副本
```

File: benchmark/benchmark-tidb-using-ch.md [hunk 20]
```diff
@@ -0,0 +1,169 @@
...
+summary: 本文介绍如何对 TiDB 进行 CH 测试。
+---
+
+# 如何对 TiDB 进行 CH 测试
+
+本文介绍如何对 TiDB 进行 CH Benchmark 测试。
+
+[CH Benchmark](https://research.tableau.com/sites/default/files/a8-cole.pdf) 是包含 [TPC-C](http://www.tpc.org/tpcc/) 和 [TPC-H](http://www.tpc.org/tpch/) 的混合负载，也是用于测试 HTAP 系统的最常见负载。
+
+在进行 CH Benchmark 测试前，推荐部署 TiDB 的 HTAP 组件 [TiFlash](/tiflash/tiflash-overview.md)。部署后，对于 TPC-C 联机交易数据，系统自动实时同步数据到 TiFlash 组件；TiDB 优化器会自动将 TPC-H 负载的 OLAP 查询下推到 TiFlash MPP 引擎进行高效执行。
```
Comment (qiancai) [inline benchmark/benchmark-tidb-using-ch.md:12]:
```suggestion
在进行 CH Benchmark 测试前，推荐部署 TiDB 的 HTAP 组件 [TiFlash](/tiflash/tiflash-overview.md)。部署 TiFlash 并[创建 TiFlash 副本](#创建-tiflash-副本)后，对于 TPC-C 联机交易数据，系统将实时同步最新的数据到 TiFlash 组件；TiDB 优化器会自动将 TPC-H 负载的 OLAP 查询下推到 TiFlash MPP 引擎进行高效执行。
```

File: benchmark/benchmark-tidb-using-ch.md (hunk 17 above)
Comment (dbsid) [reply benchmark/benchmark-tidb-using-ch.md:69]: 赞，学习了，我写了一下这种大招

File: benchmark/benchmark-tidb-using-ch.md [hunk 21]
```diff
@@ -0,0 +1,169 @@
...
+summary: 本文介绍如何对 TiDB 进行 CH 测试。
+---
+
+# 如何对 TiDB 进行 CH 测试
+
+本文介绍如何对 TiDB 进行 CH Benchmark 测试。
+
+[CH Benchmark](https://research.tableau.com/sites/default/files/a8-cole.pdf) 是包含 [TPC-C](http://www.tpc.org/tpcc/) 和 [TPC-H](http://www.tpc.org/tpch/) 的混合负载，也是用于测试 HTAP 系统的最常见负载。
+
+在进行 CH Benchmark 测试前，推荐部署 TiDB 的 HTAP 组件 [TiFlash](/tiflash/tiflash-overview.md)。部署 TiFlash 并[创建 TiFlash 副本](#创建-tiflash-副本)后，对于 TPC-C 联机交易数据，系统将实时同步最新的数据到 TiFlash 组件；TiDB 优化器会自动将 TPC-H 负载的 OLAP 查询下推到 TiFlash MPP 引擎进行高效执行。
```
Comment (qiancai) [inline benchmark/benchmark-tidb-using-ch.md:12]:
```suggestion
在进行 CH Benchmark 测试前，你需要先部署 TiDB 的 HTAP 组件 [TiFlash](/tiflash/tiflash-overview.md)。部署 TiFlash 并[创建 TiFlash 副本](#创建-tiflash-副本)后，对于 TPC-C 联机交易数据，系统将实时同步最新的数据到 TiFlash 组件；TiDB 优化器会自动将 TPC-H 负载的 OLAP 查询下推到 TiFlash MPP 引擎进行高效执行。
```

File: benchmark/benchmark-tidb-using-ch.md [hunk 22]
```diff
@@ -0,0 +1,169 @@
...
+
+本文将以 1000 WAREHOUSE 为例进行导入和测试。在 shell 中运行 TiUP 命令：
+
+{{< copyable "shell-regular" >}}
+
+```shell
+tiup bench tpcc -H 172.16.5.140 -P 4000 -D tpcc --warehouses 1000 prepare
+```
+
+基于不同的机器配置，这个过程可能会持续几个小时。如果是小型集群，可以使用较小的 WAREHOUSE 值进行测试。
```
Comment (qiancai) [inline benchmark/benchmark-tidb-using-ch.md:38]:
```suggestion
基于不同的机器配置，数据导入过程可能会持续几个小时。如果是小型集群，可以使用较小的 WAREHOUSE 值进行测试。
```

**Comment by dbsid** [comment]
@lilinghai @qiancai Please take a look again

**Review by lilinghai — APPROVED** [review]
lgtm

File: benchmark/benchmark-tidb-using-ch.md [hunk 23]
```diff
@@ -0,0 +1,171 @@
...
+[Summary] Q21    - Count: 10, Sum(ms): 166001.4, Avg(ms): 16601.1
+[Summary] Q22    - Count: 10, Sum(ms): 48268.4, Avg(ms): 4827.7
+[Summary] Q3     - Count: 11, Sum(ms): 31110.1, Avg(ms): 2828.5
+[Summary] Q4     - Count: 11, Sum(ms): 83814.2, Avg(ms): 7619.3
+[Summary] Q5     - Count: 11, Sum(ms): 368301.0, Avg(ms): 33483.5
+[Summary] Q6     - Count: 11, Sum(ms): 61702.5, Avg(ms): 5608.9
+[Summary] Q7     - Count: 11, Sum(ms): 158928.2, Avg(ms): 14446.3
+```
+
+测试完成之后，也可以运行 `tiup bench tpcc -H 172.16.5.140 -P 4000 -D tpcc --warehouses 1000 check` 进行数据正确性验证。
```
Comment (qiancai) [inline benchmark/benchmark-tidb-using-ch.md:171]:
```suggestion
测试完成之后，也可以运行 `tiup bench tpcc -H 172.16.5.140 -P 4000 -D tpcc --warehouses 1000 check`  验证数据正确性。
```

File: benchmark/benchmark-tidb-using-ch.md [hunk 24]
```diff
@@ -0,0 +1,171 @@
...
+```
+
+查询结果中：
+
+* AVAILABLE 字段表示该表的 TiFlash 副本是否可用。1 代表可用，0 代表不可用。副本状态为可用之后就不再改变，如果通过 DDL 命令修改副本数则会重新计算同步进度。
+* PROGRESS 字段代表同步进度，在 0.0~1.0 之间，1 代表至少 1 个副本已经完成同步。
+
+## 搜集统计信息
+
+为了确保优化器能产生正确的执行计划，需要提前搜集统计信息。
```
Comment (qiancai) [inline benchmark/benchmark-tidb-using-ch.md:90]:
```suggestion
为了确保优化器能生成最优的执行计划，请执行以下 SQL 语句提前搜集统计信息：
```

File: benchmark/benchmark-tidb-using-ch.md [hunk 25]
```diff
@@ -0,0 +1,171 @@
...
+{{< copyable "sql" >}}
+
+```sql
+SELECT * FROM information_schema.tiflash_replica WHERE TABLE_SCHEMA = 'tpcc';
+```
+
+查询结果中：
+
+* AVAILABLE 字段表示该表的 TiFlash 副本是否可用。1 代表可用，0 代表不可用。副本状态为可用之后就不再改变，如果通过 DDL 命令修改副本数则会重新计算同步进度。
+* PROGRESS 字段代表同步进度，在 0.0~1.0 之间，1 代表至少 1 个副本已经完成同步。
```
Comment (qiancai) [inline benchmark/benchmark-tidb-using-ch.md:86]: “1 代表” 还是”1.0 代表”呢，可以在结果中看下确认下吗？

File: benchmark/benchmark-tidb-using-ch.md [hunk 26]
```diff
@@ -0,0 +1,171 @@
...
+
+## 创建 TiFlash 副本
+
+部署 TiFlash 后，TiFlash 并不会自动同步 TiKV 数据，你需要执行以下 SQL 语句创建整库的 TiFlash 副本。创建 TiFlash 副本后，系统自动实时同步最新数据到 TiFlash 组件。以下例子中，集群中部署了两个 TiFlash 节点，将 replica 设置为 2。
+
+```
+alter database tpcc set tiflash replica 2;
+```
+
+可通过如下 SQL 语句确认所有表（通过 WHERE 语句指定，去掉 WHERE 语句则查看所有表）的 TiFlash 副本的状态是否完成同步：
```
Comment (qiancai) [inline benchmark/benchmark-tidb-using-ch.md:75]:
```suggestion
可通过如下 SQL 语句确认所有表（通过 WHERE 语句可以指定需要确认的表，去掉 WHERE 语句则查看所有表）的 TiFlash 副本的状态是否完成同步：
```

File: benchmark/benchmark-tidb-using-ch.md [hunk 27]
```diff
@@ -0,0 +1,171 @@
+---
+title: 如何对 TiDB 进行 CH 测试
+summary: 本文介绍如何对 TiDB 进行 CH 测试。
+---
+
+# 如何对 TiDB 进行 CH 测试
+
+本文介绍如何对 TiDB 进行 CH Benchmark 测试。
+
+[CH Benchmark](https://research.tableau.com/sites/default/files/a8-cole.pdf) 是包含 [TPC-C](http://www.tpc.org/tpcc/) 和 [TPC-H](http://www.tpc.org/tpch/) 的混合负载，也是用于测试 HTAP 系统的最常见负载。
```
Comment (qiancai) [inline benchmark/benchmark-tidb-using-ch.md:10]:
```suggestion
CH Benchmark 是包含 [TPC-C](http://www.tpc.org/tpcc/) 和 [TPC-H](http://www.tpc.org/tpch/) 的混合负载，也是用于测试 HTAP 系统的最常见负载。更多信息，请参考 [The mixed workload CH-benCHmark](https://research.tableau.com/sites/default/files/a8-cole.pdf)。
```

File: benchmark/benchmark-tidb-using-ch.md (hunk 25 above)
Comment (dbsid) [reply benchmark/benchmark-tidb-using-ch.md:86]: 不用改了吧，这两句是从 tiflash 文档抄过来的  https://docs.pingcap.com/zh/tidb/v3.1/use-tiflash#%E6%9F%A5%E7%9C%8B%E8%A1%A8%E5%90%8C%E6%AD%A5%E8%BF%9B%E5%BA%A6

**Review by qiancai — APPROVED** [review]
LGTM

File: TOC.md [hunk 28]
```diff
@@ -99,6 +99,7 @@
   - 测试集群性能
     - [用 Sysbench 测试 TiDB](/benchmark/benchmark-tidb-using-sysbench.md)
     - [对 TiDB 进行 TPC-C 测试](/benchmark/benchmark-tidb-using-tpcc.md)
+    - [对 TiDB 进行 CH 测试](/benchmark/benchmark-tidb-using-ch.md)
```
Comment (qiancai) [inline TOC.md:102]:
```suggestion
    - [对 TiDB 进行 CH Benchmark 测试](/benchmark/benchmark-tidb-using-ch.md)
```

File: benchmark/benchmark-tidb-using-ch.md [hunk 29]
```diff
@@ -0,0 +1,171 @@
+---
+title: 如何对 TiDB 进行 CH 测试
+summary: 本文介绍如何对 TiDB 进行 CH 测试。
```
Comment (qiancai) [inline benchmark/benchmark-tidb-using-ch.md:3]:
```suggestion
title: 如何对 TiDB 进行 CH Benchmark 测试
summary: 本文介绍如何对 TiDB 进行 CH Benchmark 测试。
```

File: benchmark/benchmark-tidb-using-ch.md [hunk 30]
```diff
@@ -0,0 +1,171 @@
...
+### 导入 TPC-C 数据
+
+**导入数据通常是整个 TPC-C 测试中最耗时，也是最容易出问题的阶段。**
+
+本文将以 1000 WAREHOUSE 为例进行导入和测试。在 shell 中运行 TiUP 命令：
+
+{{< copyable "shell-regular" >}}
+
+```shell
+tiup bench tpcc -H 172.16.5.140 -P 4000 -D tpcc --warehouses 1000 prepare -T 32
```
Comment (qiancai) [inline benchmark/benchmark-tidb-using-ch.md:35]:
```suggestion
tiup bench tpcc -H 172.16.5.140 -P 4000 -D tpcc --warehouses 1000 prepare -T 32
```

**Pull request merged**